examples/resnet-retrain/README.md

```diff
@@ -1,5 +1,9 @@
 # resnet-retrain
 
+## Snapshot version of Cortex
```

Comment:
I'll change this to non-snaphsot https://github.com/thinktopic/cortex/issues/237 to be in sync with https://github.com/thinktopic/cortex/issues/237